controllers/login.js

```diff
@@ -148,26 +149,43 @@ router.all('/login/', (req, res, next) => {
 		if (isAuthenticated) {
 			return redirectAuthenticated(req, res);
 		}
-		return authHelper.clearCookie(req, res)
-			.then(() => getSelectOptions(req,
-				'schools', {
-					purpose: { $ne: 'expert' },
-					$limit: false,
-					$sort: 'name',
-				})
-				.then((schools) => {
-					res.render('authentication/login', {
-						schools,
-						systems: [],
-						redirect: req.query && req.query.redirect ? req.query.redirect : '',
-					});
-				}));
+		return handleLoginFailed(req,res);
 	}).catch((error) => {
 		logger.error('Error during login', { error: error.toString() });
 		return next(error);
 	});
 });
 
+router.all('/login/superhero/', (req, res, next) => {
+	res.locals.notification = {
+		type: 'danger',
+		message: res.$t('login.text.superheroForbidden'),
+		statusCode: 401,
+		timeToWait: LOGIN_BLOCK_TIME || 15,
```

Comment:
use configuration instead, otherwise we facing issues here with number or string type. default has to be moved to globals (everything to configuration schema)

Reply:
ok, resolved